middleware/downloadcsv.js

```diff
@@ -1,18 +1,55 @@
 const { parse, transforms: { unwind } } = require("json2csv")
 
+const get_qcode = (row) => {
+  if (row.answers.options && row.answers.options.q_code) {
+    return row.answers.options.q_code
+  }
+  if (row.answers.options && row.answers.options.qCode) {
+    return row.answers.options.qCode
+  }
+  if (row.answers.q_code) {
+    return row.answers.secondaryQCode ? `${row.answers.q_code} / ${row.answers.secondaryQCode}` : row.answers.q_code
+  }
+  if (row.answers.qCode) {
+    return row.answers.secondaryQCode ? `${row.answers.qCode} / ${row.answers.secondaryQCode}` : row.answers.qCode
+  }
+}
+
 const fields = [
-  "id",
-  "title",
-  "answers.id",
-  "answers.label",
-  "answers.options.label",
-  "answers.options.value"
+  {
+    label: "Author short code",
+    value: (row) => row.alias
+  },
+  {
+    label: "Question",
+    value: (row) => row.title
+  },
+  {
+    label: "Type",
+    value: (row) => row.answers.type
+  },
+  {
+    label: "Answer label",
+    value: (row) => row.answers.secondaryLabel ? `${row.answers.label} / ${row.answers.secondaryLabel}` : row.answers.label
+  },
+  {
+    label: "QCode",
+    value: (row) => get_qcode(row)
+  },
+  {
+    label: "Options label",
+    value: (row) => row.answers.options ? row.answers.options.label : ""
+  }
 ]
 
-const transforms = [unwind({ paths: ["answers", "answers.options"] })]
+const transforms = [unwind({ paths: ["answers", "answers.options"], blankOut: true })]
 
 module.exports = (req, res) => {
   const data = parse(JSON.parse(req.body.questions), { fields, transforms })
   res.attachment("questions.csv")
-  res.send(data)
+
+  const parsedText = data
+    .replace(/(<([^>]+)>)/gi, "")
+    .replace("&#x27;", "'")
```

Comment:
Tested and working Josh - just spotted one bug though which is that only the first apostrophe gets unescaped with the replace() as it is. Need to change to global-searching regex (like with the tags):

```suggestion
    .replace(/&#x27;/gi, "'")
```

Reply:
There is already a prewritten function for formatting the question title which is more thorough than the regex there. 
You could apply it against the title field directly.